skpro/tests/test_polars.py

```diff
@@ -0,0 +1,121 @@
+"""Test file for polars dataframes"""
+
+import pandas as pd
+import polars as pl
+import pytest
+from sklearn.datasets import load_diabetes
+from sklearn.model_selection import train_test_split
+
+from skpro.datatypes import check_raise
+from skpro.datatypes._table._check import check_polars_table
+from skpro.datatypes._table._convert import convert_pandas_to_polars_eager
+from skpro.utils.validation._dependencies import _check_soft_dependencies
+
+X, y = load_diabetes(return_X_y=True, as_frame=True)
+X = X.iloc[:75]
+y = y.iloc[:75]
+
+# typically y is returned as a pd.Series to we call y as a Dataframe here
+y = pd.DataFrame(y)
+
+X_train, X_test, y_train, _ = train_test_split(X, y, test_size=0.33, random_state=42)
+
+TEST_ALPHAS = [0.05, 0.1, 0.25]
+
+
+@pytest.mark.skipif(
+    not _check_soft_dependencies(["polars", "pyarrow"], severity="none"),
+    reason="skip test if polars/pyarrow is not installed in environment",
+)
+def test_polars_eager_conversion_methods():
```

Comment:
possibly redundant - these should already be tested by various tests in the `datatypes` module, have you looked whether this is the case?

Reply:
Yes I think they are covered inside `datatypes.tests.test_convert.py`?